### PR TITLE
darktile: 0.0.10 -> 0.0.11

### LINKED ISSUE #220883: Update request: darktile 0.0.10 → 0.0.11
- Package name: darktile
- Latest released version: 0.0.11

- Current version on the unstable channel: 0.0.10
- Current version on the stable/release channel: 0.0.10

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**
@flexagoon

-----

Note for maintainers: Please tag this issue in your PR.

### PR DESCRIPTION
**Pull request by MikaelFangel**
## Description of changes
 * Replaces the maintainer as, whished [here](https://github.com/NixOS/nixpkgs/issues/220883#issuecomment-1465609453)
 * Closes #220883 and solves #221013
 * Change to use buildGoModule instead of stdenv.mkDerivation

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by kirillrdy** [comment]
@MikaelFangel please resolve merge conflict

File: pkgs/applications/terminal-emulators/darktile/default.nix [hunk 1]
```diff
@@ -12,20 +11,23 @@
...
   pname = "darktile";
-  version = "0.0.10";
+  version = "0.0.11";
 
   src = fetchFromGitHub {
     owner = "liamg";
     repo = "darktile";
     rev = "v${version}";
-    sha256 = "0pdj4yv3qrq56gb67p85ara3g8qrzw5ha787bl2ls4vcx85q7303";
+    sha256 = "sha256-M3vySAyYwqscR9n0GGXp1ttO/mhdSCponZNYJRBBI18=";
```
Comment (kirillrdy) [inline pkgs/applications/terminal-emulators/darktile/default.nix:25]:
```suggestion
    hash = "sha256-M3vySAyYwqscR9n0GGXp1ttO/mhdSCponZNYJRBBI18=";
```

**Comment by kirillrdy** [comment]
change log https://github.com/liamg/darktile/releases/tag/v0.0.11

**Review by kirillrdy — APPROVED** [review]
works on x86_64-linux

but on aarch64-linux I get

```
./result/bin/darktile 
GLFW: An uncaught error has occurred: APIUnavailable: GLX: No GLXFBConfigs returned
                                                                                   GLFW: Please report this bug in the Go package immediately.
                                                                                                                                              GLFW: An invalid error was not accepted by the caller: FormatUnavailable: GLX: Failed to find a suitable GLXFBConfig
                                             GLFW: Please report this bug in the Go package immediately.
                                                                                                        panic: FormatUnavailable: GLX: Failed to find a suitable GLXFBConfig

                                                                                                                                                                            goroutine 1 [running, locked to thread]:
                                                                                                                                                                                                                    github.com/go-gl/glfw/v3.3/glfw.acceptError({0x40002c9830, 0x2, 0x40002c9828?})
                                                                              	github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20210727001814-0db043d8d5be/error.go:178 +0x1d8
                                                                                                                                                                      github.com/go-gl/glfw/v3.3/glfw.CreateWindow(0x4000000001?, 0x4ae490?, {0x0?, 0x105a900?}, 0x0, 0x0)
                                                     	github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20210727001814-0db043d8d5be/window.go:350 +0x144
                                                                                                                                               github.com/hajimehoshi/ebiten/v2/internal/glfw.CreateWindow(0x9?, 0x40002c98e8?, {0x0?, 0x688d08?}, 0x0?, 0x0?)
                                         	github.com/hajimehoshi/ebiten/v2@v2.2.0-alpha.11.0.20210724070913-1706d9436a78/internal/glfw/glfw_notwindows.go:257 +0x48
                                                                                                                                                                         github.com/hajimehoshi/ebiten/v2/internal/uidriver/glfw.(*UserInterface).createWindow(0xb03f40)
                                                   	github.com/hajimehoshi/ebiten/v2@v2.2.0-alpha.11.0.20210724070913-1706d9436a78/internal/uidriver/glfw/ui.go:745 +0x40
                                                                                                                                                                             github.com/hajimehoshi/ebiten/v2/internal/uidriver/glfw.(*UserInterface).init(0xb03f40)
                                               	github.com/hajimehoshi/ebiten/v2@v2.2.0-alpha.11.0.20210724070913-1706d9436a78/internal/uidriver/glfw/ui.go:878 +0x140
                                                                                                                                                                      github.com/hajimehoshi/ebiten/v2/internal/uidriver/glfw.(*UserInterface).Run.func1.2()
                                       	github.com/hajimehoshi/ebiten/v2@v2.2.0-alpha.11.0.20210724070913-1706d9436a78/internal/uidriver/glfw/run_notsinglethread.go:47 +0x20
                                                                                                                                                                             github.com/hajimehoshi/ebiten/v2/internal/thread.(*OSThread).Loop(0x4000012180)
                                       	github.com/hajimehoshi/ebiten/v2@v2.2.0-alpha.11.0.20210724070913-1706d9436a78/internal/thread/thread.go:51 +0x60
                                                                                                                                                         github.com/hajimehoshi/ebiten/v2/internal/uidriver/glfw.(*UserInterface).Run(0xb03f40, {0x874f68?, 0x1059fc0?})
                                                   	github.com/hajimehoshi/ebiten/v2@v2.2.0-alpha.11.0.20210724070913-1706d9436a78/internal/uidriver/glfw/run_notsinglethread.go:60 +0x1c0
                                                                                                                                                                                              github.com/hajimehoshi/ebiten/v2.RunGame({0x873f28?, 0x40000fa000})
                                            	github.com/hajimehoshi/ebiten/v2@v2.2.0-alpha.11.0.20210724070913-1706d9436a78/run.go:158 +0xd4
                                                                                                                                               github.com/liamg/darktile/internal/app/darktile/gui.(*GUI).Run(0x40000fa000)
      	github.com/liamg/darktile/internal/app/darktile/gui/gui.go:99 +0x1b8
                                                                            github.com/liamg/darktile/internal/app/darktile/cmd.glob..func2(0x40000c2200?, {0x7d3f83?, 0x4?, 0x7d3f87?})
                                                                                                                                                                                        	github.com/liamg/darktile/internal/app/darktile/cmd/root.go:130 +0xbd4
                                                 github.com/spf13/cobra.(*Command).execute(0xae9640, {0x4000210080, 0x0, 0x0})
                                                                                                                              	github.com/spf13/cobra@v1.1.3/command.go:852 +0x554
                                                                                                                                                                                   github.com/spf13/cobra.(*Command).ExecuteC(0xae9640)
                  	github.com/spf13/cobra@v1.1.3/command.go:960 +0x310
                                                                           github.com/spf13/cobra.(*Command).Execute(...)
                                                                                                                         	github.com/spf13/cobra@v1.1.3/command.go:897
                                                                                                                                                                            github.com/liamg/darktile/internal/app/darktile/cmd.Execute()
                    	github.com/liamg/darktile/internal/app/darktile/cmd/root.go:153 +0x224
                                                                                              main.main()
                                                                                                         	github.com/liamg/darktile/cmd/darktile/main.go:75 +0x1c
```

maybe its something with qemu, I can try tomorrow on real device, but no need to block this PR

**Comment by MikaelFangel** [comment]
It actually had I similar bug on x86-64 until I rebased with master, so I'm a bit unsure where this comes from. 

As an alternative we could use the platform meta tags?

**Comment by kirillrdy** [comment]
> It actually had a similar bug on x86-64 until I rebased with master, so I'm a bit unsure where this comes from.
> 
> As an alternative we could use the platform meta tags?

At this stage I wouldn't worry. If you want to wait for me to try on real deice, otherwise I assume its working on aarch64-linux

**Comment by MikaelFangel** [comment]
Sure, I'll wait (:

**Comment by kirillrdy** [comment]
> Sure, I'll wait (:

doesn't work either.

image

maybe disable for aarch64-linux and leave a comment to try again at next release

**Pull request merged**